### DISCUSSION TITLE
How to use the collect WL?

### DISCUSSION THREAD

**Discussion by Airunn** [thatskymod/Sky-CotL-Scripts#50] [General]
I'm confused on how to use the collect WL, it doesn't seem to do anything for me???

**Comment by thatskymod** [comment]
Alright, @Airunn!

It looks like there's a small mistake with the way you're using the WL collector. Let’s sort it out!

Each winged light in Sky has its own unique codename or `id`. Here's what they look like:

![0522d0d6-75d8-4cbb-b7c5-e6bef4933767](https://github.com/thatskymod/Sky-CotL-Scripts/assets/90126520/45b9ce9d-88b8-4b1a-b99f-295c6ad5a2fc)
`----------rest trimmed for brevity----------`

And the list just keeps going! As a result, you need to specify the particular wing `id`s you're after. Rest assured, there's no need to memorize a single thing.

We've got a wing list ready for you to grab and pop right into the WL collector prompt: [236_wing_code_thatskymod.txt](https://raw.githubusercontent.com/thatskymod/Sky-CotL-Scripts/main/Canvas/Misc/236_wing_code_thatskymod.txt).

You’ve got a couple of options:
- Copy the entire list to clipboard and press "Paste" in the WL collector.
- You can also use your browser's download function to save it as a local file and later load it into AutoWax.

Once you’re all set, hit "Run" to finalize the collection. For the increased wing level to take effect, please restart your game.
